operations/gettingstarted/README.md

```diff
@@ -16,41 +16,48 @@ or HTTPS:
 ```
 git clone https://{yourusername}@bitbucket.org/livestyled-dev/frontier.ios.git
 ```
-
-## Installing dependencies
+## Perform Pre-Build Configuration
 
 ### Requirements
-- Permission to access the [Kaltura repository](https://livestyledios@bitbucket.org/livestyled-dev/kaltura-ios-sdk.git)
-- The [CocoaPods CLI](https://cocoapods.org)
+* If you wish to build a client's app, you will need access to [Frame](https://frame.realifetech.com/), or to know the client's app code.
 
 ### Instructions
-Inside the project's directory, run:
+
+Navigate to the project's scripts directory
 ```
-pod install
+cd ./scripts
 ```
-If you run into issues when switching between brances with different pods, you may need to run:
+Run the fetch assets script
 ```
-pod install --repo update
+bash fetch_assets.sh
 ```
 
-## Switching between branches/apps
-Fetch all branches using:
+The above a command will fetch the platform (or default) configuration for the app. If you wish to build for a specific client (using their feature activations, styles, endpoints etc), look up their 'app code' on Frame and pass it into the fetch assets script:
 ```
-git fetch
+bash fetch_assets.sh APP_CODE
 ```
-Check out a branch using:
+
+## Installing dependencies
+
+### Requirements
+- Permission to access the [Kaltura repository](https://livestyledios@bitbucket.org/livestyled-dev/kaltura-ios-sdk.git)
+- The [CocoaPods CLI](https://cocoapods.org)
+
+### Instructions
+Inside the project's root directory, run:
 ```
-git checkout {branchname}
+pod install
 ```
-Reset changes using (you should do this when finished, and when switching branches):
+If you run into issues when switching between brances with different pods, you may need to run:
 ```
-git checkout .
+pod install --repo update
 ```
+Note, if you run the install command from outside the root, you may see the following error: `Invalid Podfile file: cannot load such file -- Scripts/DynamicModules.rb`. To fix this, in your terminal navigate back to the project root and run `$ pod install` again.
 
 ## Running the project
 
 ### Requirements
-- XCode 10.2.1
+- XCode 12.4
```

Comment:
How about updating `XCode` to `Xcode` please?

Reply:
✅